apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -111,6 +113,32 @@ private function register_patterns() {
 	 * @return array
 	 */
 	private function get_patterns() {
+		if ( defined( 'WPCOM_PATTERNS_OVERRIDE_SOURCE_SITE' ) && WPCOM_PATTERNS_OVERRIDE_SOURCE_SITE ) {
```

Comment:
In general we've been trying to avoid putting Dotcom specific code into this plugin, since it ships to .org plugin repo too.

Reply:
I've just updated this to use a filter instead, thanks for the suggestion!